### ISSUE TITLE
Pruning CTRL (lower compute/memory usage)

### ISSUE THREAD

**Issue by Kampffussel03** [tusen-ai/SST#194]
Hi Lue,

many thanks again for your great work. As CTRL_FSD was trained with much compute, what would be the best cuts for a still high-performing, pruned version? Do you have any quick hints or ideas about that @Abyssaledge ?

I noticed that FrameDropout is not used and was wondering why. I was looking for the implementation of the following modifications 2) and 3) mentioned here in your paper:
![image](https://github.com/tusen-ai/SST/assets/123517225/d1ee3964-b2e7-4abf-9fc2-c038a209923e)

Maybe a quick modification to test would be using less frames, e.g. 5 instead of 9, or a higher dropout if that's worth in terms of compute. Have you tried something already in this regard?

Best regards

**Comment by Kampffussel03** [comment]
The number of frame for actual training is set in the dataloader class.

For pruning CTRL, you can just use a small backbone, which does not decrease the performance too much.
